src/domain/app/App.tsx

```diff
@@ -91,6 +92,14 @@ const App: React.FunctionComponent = props => {
           <ProfileRoute />
         </PrivateRoute>
 
+        {
+          // TODO: The route should probably be /profile/child/${child.id}/events/${event.id}
+          // we're waiting for KK-277.
+        }
+        <PrivateRoute path={`/${locale}/event`}>
+          <Event />
```

Comment:
can user who doesnt have profile able to see this?

Reply:
Based on what I know this should be a private route. Only logged in people can be invited and sign up for occurrences.